common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -350,7 +351,7 @@ def get_html(self):
             'cdn_eval': cdn_eval,
             'cdn_exp_group': cdn_exp_group,
             'id': self.location.html_id(),
-            'display_name': self.display_name_with_default_escaped,
+            'display_name': self.display_name_with_default,
```

Comment:
@robrap Thanks for the heads up!

That [block of code](common/lib/xmodule/xmodule/video_module/video_module.py) renders to [lms/templates/video.html](https://github.com/open-craft/edx-platform/blob/33b3dfcc9279ce59a1b8adf49adb638202d8680f/lms/templates/video.html), which has `<%page expression_filter="h"/>`, so does that mean `display_name` is escaped?